index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" class="header-logo">Mesfin</a>
+      <nav class="header-menu-btn mobile">
+        <div class="menu-btn__ham one"></div>
+        <div class="menu-btn__ham two"></div>
+        <div class="menu-btn__ham three"></div>
```

Comment:
- [ ] [OPTIONAL] You should strongly consider using li tags for these and wrapping them in a ul tag